### ISSUE TITLE
Gained an advancement when starting a game

### ISSUE THREAD

**Issue by linzexin022** [TelepathicGrunt/RepurposedStructures-Fabric#26]
Minecraft Version: 1.16.2
Mod Version: 1.5.2
Installed other mods?: A lot

Hi there, I found I gained an advancement "Twisted Townsfolks" when starting a game. I installed a lot of other mods, for example, _Origins_, which keeps me invincible at choosing starting race before I enter the world (maybe transport to a strange place). I tested your mod seperately and with mod _Origins_, found it happened only with mod _Origins_. I wonder if you can communicate with Origins author Apace100 to solve this problem?

![image](https://user-images.githubusercontent.com/59003232/91799813-67265a80-ec5a-11ea-90f3-2d74fa7da7e3.png)

**Comment by TelepathicGrunt** [comment]
what in the world???
Can you open your advancement screen and see if that's the only repurposed structures advancement activated? 

My advancements are done using purely vanilla json stuff and should only activate in the warped village which is why this massively odd. Hence why I ask to see if it's just that advancement or others
<https://github.com/TelepathicGrunt/RepurposedStructures-Fabric/blob/master/src/main/resources/data/repurposed_structures/advancements/village_warped.json>

**Comment by Afrocntrik** [comment]
I was about to report it as well it's just that advancement on world spawn
![2020-09-01_20 06 04](https://user-images.githubusercontent.com/11052983/91942759-742e6280-ecc9-11ea-8963-95375a73b71d.png)

**Comment by TelepathicGrunt** [comment]
Update: I got the bug to occur even without other mods. very odd. Still looking into it

**Comment by TelepathicGrunt** [comment]
I can't seem to figure out why only this one village advancement is breaking and not for others. For now, you can use this datapack to disable that advancement so it doesn't pop up all the time while I look for a solution

[Repurposed_Structures-Disabled_Warped_Village_Advancement.zip](https://github.com/TelepathicGrunt/RepurposedStructures-Fabric/files/5163191/Repurposed_Structures-Disabled_Warped_Village_Advancement.zip)

**Comment by TelepathicGrunt** [comment]
Fixed in 1.5.3! turned out when I switched to Fabric API's way of registering structures, I accidentally left off a method at the end of a line that would've registered Warped Villages. What makes it even worse is that a structure based advancements doesn't error when a structure is missing but just rewards the advancement right away wtf. 

Now Warped Villages are registered properly and should generate again!
https://www.curseforge.com/minecraft/mc-mods/repurposed-structures-fabric/files/3048257
